script/entrypoint.sh

```diff
@@ -2,23 +2,9 @@
 set -ex
 
 # Render a AWS credentials with user and role for `awsudo`.
-DEFAULT_REGION=us-east-1
+DEFAULT_REGION=ap-southeast-1
```

Comment:
Is default region required ?

Reply:
option to override region is also present